packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -258,19 +252,12 @@ function BaseLightningElementConstructor(this: LightningElement) {
         mode,
         def: { ctor },
     } = vm;
+
     const component = this;
     vm.component = component;
     vm.tro = getTemplateReactiveObserver(vm);
     vm.oar = create(null);
-    // interaction hooks
```

Comment:
@pmdartus we can't remove this atm. Locker relies on the ability to pass that first argument to provide its own hooks instead of using our default implementation. It is an injection mechanism at this point, something that should not be needed in the future with the locker vnext.

Reply:
Ok I will add a test for this then. No test is failing with this removal.